.github/workflows/tests-l4lb.yaml

```
@@ -0,0 +1,69 @@
+name: Cilium L4LB XDP
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  issue_comment:
+    types:
+      - created
+  # Run every 6 hours
+  schedule:
+    - cron:  '0 4/6 * * *'
```

[Nit] The 4th slot is already used by the multicluster workflow, suggesting 5th to balance out actions a little bit:
```suggestion
    - cron:  '0 5/6 * * *'
```